doc/api/events.md

```diff
@@ -393,6 +413,8 @@ By default, event listeners are invoked in the order they are added. The
 event listener to the beginning of the listeners array.
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of a previous example), that means that this is not the correct place for `use strict`.